boards/riscv/icev_wireless/icev_wireless_defconfig

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_BOARD_ICEV_WIRELESS=y
+CONFIG_SOC_ESP32C3=y
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=1000000
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_SERIAL_ESP32_USB=y
```

Comment:
CONFIG_SERIAL_ESP32_USB should get enabled by default from Kconfig & dts now.

Reply:
Fixed!